es-vue-base/src/lib-components/EsFileUpload.vue

```diff
@@ -0,0 +1,254 @@
+<!-- eslint-disable vuejs-accessibility/no-static-element-interactions -->
+<template>
+    <div
+        class="es-file-upload"
+        :class="{ 'active': active, 'inactive': !active}"
+        @dragenter.stop.prevent="setActive"
+        @dragover.stop.prevent="setActive"
+        @dragleave.stop.prevent="setInActive"
```

Comment:
You can modify inline data without a method.
```suggestion
        @dragenter.stop.prevent="active = true"
        @dragover.stop.prevent="active = true"
        @dragleave.stop.prevent="active = false"
```